node/cn/protocol.go

```diff
@@ -124,6 +125,32 @@ type txPool interface {
 	SubscribeNewTxsEvent(chan<- blockchain.NewTxsEvent) event.Subscription
 }
 
+type blockChain interface {
+	Genesis() *types.Block
```

Comment:
This interface will be used in next PR for unit test? :)

Reply:
After replacing some structs in `ProtocolManager`, I will start adding tests of `ProtocolManager`